### PR TITLE
Minor debug changes.

### PR DESCRIPTION
**Pull request by manjujames**

### CODE REVIEW

File: src/frontend/org/voltdb/exportclient/SocketExporter.java [hunk 1]
```diff
@@ -255,6 +255,9 @@ public boolean processRow(ExportRow rd) throws ExportDecoderBase.RestartBlockExc
                     m_logger.rateLimitedLog(120, Level.ERROR, null, "Failed to connect to export socket endpoint %s, some servers may be down.", host);
                     throw new RestartBlockException(true);
                 }
+                if (m_logger.isDebugEnabled()) {
+                    m_logger.warn(m_source.tableName + ":P" + m_source.partitionId + " sending seqNum: " + rd.values[2]);
```
Comment (Chaozz) [inline src/frontend/org/voltdb/exportclient/SocketExporter.java]: log-level: warn -> debug?

Comment (manjujames) [reply src/frontend/org/voltdb/exportclient/SocketExporter.java]: Yes, thanks!

File: tests/frontend/org/voltdb/export/SocketExportTestServer.java [hunk 2]
```diff
@@ -144,6 +145,14 @@ public void verifyExportedTuples(int expsize, long waitTimeMs) {
...
         System.out.println("Seen Id size is: " + m_seenIds.size() + " expected:" + expsize + " Passed: " + passed);
+        if (!passed) { // Write more debug info if it failed
+            long total = 0;
+            for (AtomicLong al : m_seenIds.values()) {
+                total += al.longValue();
+            }
+            System.out.println("Found total from values: " + total);
+            System.out.println("keys: " + new TreeSet<Long>(m_seenIds.keySet()));
+        }
         assertTrue(passed);
```
Comment (n3world) [inline tests/frontend/org/voltdb/export/SocketExportTestServer.java]: Minor: You could just make the above string the message to the `fail()` method

Comment (manjujames) [reply tests/frontend/org/voltdb/export/SocketExportTestServer.java]: There is some amount of computation required, which we don't need to do when it passes, which is most of the time.

**Pull request merged**